content/javascript/concepts/arrays/terms/indexOf/indexOf.md

```diff
@@ -48,7 +46,7 @@ console.log(indexOfPineapple);
 // Output: -1
 ```
 
-Check if color `'blue'` is in the `colors` array and after the first element:
+Check if color `'blue'` is in the `colors` array and after the second element:
```

Comment:
```suggestion
Check if color `'blue'` is in the `colors` array starting with the second element:
```

Reply:
@SSwiniarski thank you! I was trying to think of a way to make that clearer :)